packages/config/config/devices/0x0312/mp22z.json

```diff
@@ -8,104 +8,79 @@
 			"productType": "0xff00",
 			"productId": "0xff07",
 			"zwaveAllianceId": 3711
-		},
-		{
-			"productType": "0xff07",
-			"productId": "0xff03",
-			"zwaveAllianceId": 4254
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"associations": {
-		"1": {
-			"label": "Lifeline",
-			"maxNodes": 3,
-			"isLifeline": true
-		},
-		"2": {
-			"label": "Basic Set",
-			"maxNodes": 5
-		},
-		"3": {
-			"label": "Basic Set",
-			"maxNodes": 5
-		}
-	},
```

Comment:
According to the linked manual, this should have 2 groups (Lifeline and Basic Set). Not sure if we need them to be defined, but the pre-programmed labels are usually pretty bad.